src/calendars/czechRepublic.ts

```diff
@@ -81,12 +109,9 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
       rank: Ranks.OPT_MEMORIAL,
       date: dayjs.utc(`${year}-7-4`),
     },
-    // In Slovakia and Czech Republic, the two brothers were originally
-    // commemorated on 9 March, but Pope Pius IX changed this date to 5 July
-    // https://en.wikipedia.org/wiki/Saints_Cyril_and_Methodius
     {
       key: 'cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe',
-      rank: Ranks.SOLEMNITY,
+      rank: Ranks.SOLEMNITY, // TODO: Should we add `Patrons of Moravia`?
```

Comment:
@igneus, I am not sure if it should be celebrated with patronage title of `Copatrons of Europe` or `Copatrons of Europe and Patrons of Moravia` (as it is listed in [`czech-cs.txt`](https://github.com/igneus/calendarium-romanum/blob/4918754ca64d501e3c62165d1512adbbabcc9148/data/czech-cs.txt#L122)). I remind you I need the celebration metadata as it is celebrated in the particular calendar for Czech Republic, not for Dioceses nor Bohemia nor Moravia.